tests/layer_tests/tensorflow_tests/test_tf_BatchMatrixInverse.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestBatchMatrixInverse(CommonTFLayerTest):
+    def prepare_input(self, inputs_info):
+        assert 'x:0' in inputs_info, "Test error: inputs_info must contain `x`"
+        x_shape = inputs_info['x:0']
+        inputs_data = {}
+        inputs_data['x:0'] = self._generate_invertible_matrices(x_shape)
+        return inputs_data
+    
+    def _generate_invertible_matrices(self, input_shape):
+        if input_shape == [2, 3, 3]:
+            return np.array([
+                [[1.0, 2.0, 3.0],
+                 [0.0, 1.0, 4.0],
+                 [5.0, 6.0, 0.0]],
+                [[2.0, 0.0, 0.0],
+                 [3.0, 4.0, 0.0],
+                 [5.0, 6.0, 7.0]]
+            ])
```

Comment:
`astype` is required. See a comment below

Reply:
Done